### PR TITLE
Update panel shows latest version message as success

### LINKED ISSUE #1586: upgrade check should not warn if there is no new version available
In FreshRSS, when someone check if a new version is available, the answer is displayed as a warning (in red) even if there is no new version available.

screenshot

If there is no new version available, the version used is the last one. Hence it should be a green answer, not a red one.

### PR DESCRIPTION
**Pull request by kevinpapst**
This PR changes the display of the "you are running the latest version" message.

Currently it shows that message as "alert-error" which was not only irritating to me, see also #1586.
My PR changes that message to "alert-success" to indicate a positive state. 

FIXES #1586

Further enhancement idea:
Change the wording from "Blast! No update to apply" to "Congratulations, you are running the latest version" - IMO its always better to give a positive message, but thats not part of this PR.

### CODE REVIEW

File: app/views/update/index.phtml [hunk 1]
```diff
@@ -14,7 +14,21 @@
 	</p>
 
 	<?php if (!empty($this->message)) { ?>
-	<p class="alert <?php echo $this->message['status'] === 'bad' ? 'alert-error' : 'alert-warn'; ?>">
+    <?php
```
Comment (Frenzie) [inline app/views/update/index.phtml]: Shouldn't these be tabs?

**Comment by Frenzie** [comment]
Exaltations and celebrations your liege, you are running the latest version!

/me was reminded of Civ

File: app/views/update/index.phtml (hunk 1 above)
Comment (kevinpapst) [reply app/views/update/index.phtml]: Seems so, looked fine over here. Guess I have to change my IDE settings

**Comment by kevinpapst** [comment]
Pushed an tab'ed update

**Review by Frenzie — APPROVED** [review]
lgtm

**Pull request merged**